### ISSUE TITLE
Unable to add more inner blocks to a quote block once a list inner block is added, using normal keyboard strokes

### ISSUE THREAD

**Issue by jpollock** [WordPress/gutenberg#44625]
### Description

Once I add a List block to a Quote block, I am unable to add other blocks, to the Quote block, using keyboard strokes, e.g. enter or arrow up or down. My expectation is that I would be able to use similar keystrokes, to get to add new inner blocks. The attached gif shows the issue best.

### Step-by-step reproduction instructions

1. Create a new post;
2. Add a Quote block;
3. Within the Quote block, add a List block;
4. Type some text, to add an item to the list;
5. There is no immediately obvious way to add another, as a peer to the list, within the quote block;
6. If one properly selects the list block, in its entirety, then one can place mouse in the "Add citation" section and then use the "Add Block" button.

Expectation is that there would be a more apparent way to do that, either through navigation or buttons.

### Screenshots, screen recording, code snippet

![issue_with_list_block_in_quote_block](https://user-images.githubusercontent.com/656561/193344012-9d881c79-598c-4b3f-a738-ad6d044802e1.gif)

### Environment info

WordPress version: WordPress 6.1-beta2 (running in Local)
Gutenberg version: N/A
Are all plugins except Gutenberg deactivated? No; using WordPress Beta Tester plugin
Are you using a default theme (e.g. Twenty Twenty-One)? Twenty Twenty-Three
OS: macOS Monterey 12.6
Browser: tested in Chrome, Firefox and Safari

### Please confirm that you have searched existing issues in the repo.

Yes

### Please confirm that you have tested with all plugins deactivated except Gutenberg.

Yes

**Comment by benridane** [comment]
It was able to reproduce it.
Gutenberg version: 14.2.0

It was NOT able to reproduce it.
Gutenberg version: 13.X

It was also reproduced in Group blocks.

**Comment by benridane** [comment]
It found out  that canOutdent process checked only grandParent is exists  or not. 
In this case,  parentBlock inside of GroupBlock is always true. 

List Item should only move when Grand Parent is List Item. 

https://github.com/WordPress/gutenberg/blob/d74a06f9f9c8f58eb7d49ed99e299215e44cf147/packages/block-library/src/list-item/hooks/use-outdent-list-item.js#L23-L28
